src/main/java/Domain/Users/TeamOwner.java

```diff
@@ -8,6 +8,7 @@
 public class TeamOwner extends Role {
 
     private List<Team> ownedTeams;
+    private SystemUser appointedOwner;
```

Comment:
Please add a comment to explain what this field represents.
something like: "The user who appointed this TeamOwner